build/media-suite_compile.sh

```diff
@@ -2424,6 +2426,75 @@ if [[ $ffmpeg != no ]]; then
     fi
 fi
 
+_check=(libde265.a)
+[[ $standalone = y ]] && _check+=(bin-video/dec265.exe)
+if [[ $libheif = y ]] &&
+    do_vcs "$SOURCE_REPO_LIBDE265"; then
+    do_uninstall "${_check[@]}"
+    extracommands=()
+    [[ $standalone = n ]] && extracommands+=(-DENABLE_{DE,EN}CODER=OFF)
+    do_cmakeinstall video "${extracommands[@]}"
+    do_checkIfExist
+fi
+
+_check=(bin-video/heif-{dec,enc,info,thumbnailer}.exe)
+if [[ $libheif = y ]] &&
+    do_vcs "$SOURCE_REPO_LIBHEIF"; then
+    do_uninstall "${_check[@]}"
+
+    sed -i 's/find_package(vvdec 2.3.0)/find_package(vvdec 3.0.0)/' CMakeLists.txt
+    sed -i 's/find_package(Doxygen)/#/' CMakeLists.txt # no configurable option?
+    sed -i 's/find_package(Brotli)/#/' CMakeLists.txt # linking difficulties
+    sed -i 's/find_package(TIFF)/#/' heifio/CMakeLists.txt # configure & linking difficulties
+
+    extracflags=()
+    extracommands=(-DWITH_HEADER_COMPRESSION=ON -DWITH_UNCOMPRESSED_CODEC=ON)
+    pc_exists "libde265" &&
+        extracommands+=(-DWITH_LIBDE265=ON -DWITH_LIBDE265_PLUGIN=OFF) &&
```

Comment:
I haven't looked at upstream yet, what's the deal with the `PLUGIN=OFF` stuff?

Reply:
Some of the codecs will be built as a plugin dll, even with BUILD_SHARED_LIBRARIES=OFF. But the plugin themselves don't have extra dependencies though.

Reply:
Understood